app/schema/crashes.ts

```diff
@@ -0,0 +1,61 @@
+import { z } from "zod";
+import { lookupOptionSchema } from "./lookupTable";
+import { unitSchema } from "./units";
+import { changeLogEntrySchema } from "./changeLog";
+import { crashInjuryMetricsSchema } from "./crashInjuryMetrics";
+
+export const crashSchema = z.object({
```

Comment:
I was able to test this schema and get a console error when I modified one of these column names to something that doesn't exist. I wasn't able to get feedback when I removed a column completely which I though was unexpected but now I'm not sure. 

I think this could end up with us having silent errors. If you remove `case_id` from this Zod object, no error occurs but the value no longer passes through to the Crash page.